e2e/tests/init.test.ts

```diff
@@ -42,4 +42,20 @@ describe('Init command', () => {
       ]
     `);
   });
+
+  it('should throw an error if the directory is not empty', async () => {
+    const project = new TestProject();
+
+    await execaCommand(`pnpm -s wxt init ${project.root} -t vue --pm npm`, {
+      env: { ...process.env, CI: 'true' },
+      stdio: 'ignore',
+    });
```

Comment:
```suggestion
    await fs.writeJson(project.resolve('package.json'), {});
```

Init E2E tests are slow, so rather than bootstrapping twice, creating a single file in the directory would speed them up.

Specifically, I've had problems with init tests timing out in GitHub actions when ran on windows. The install step is super slow.

Reply:
Makes sense.  

However, the `project.resolve` method does not exist, I replaced it with `project.resolvePath`. And there is still a problem after the replacement, `fs.writeJSON` will throw an error, `No such file...`

Reply:
Oh yeah, it's `resolvePath`. The no such file error was because the project directory didn't exist yet.